### ISSUE TITLE
[Bug]: Cannot scroll the offline progress toast if the list of offline progress is too long (Android Beta)

### ISSUE THREAD

**Issue by DarkPulse** [MelvorIdle/melvoridle.github.io#3480]
### Describe the bug

I upgraded to the new beta on Android, and while I'm very pleased with the performance, I found a minor, but annoying, bug: if the list of things that happened while offline is too long, it will move past the bottom of the screen.

Normally that'd be simple - swipe to scroll. However, for some reason, you cannot scroll. Therefore, you cannot get to the button needed to clear the toast. As a result, you can do no actions, and are forced to relaunch the game - by which point, the local save will have produced a much shorter list of "offline" actions, which will easily let you proceed on.

I will note that while testing this, I _**DID**_ manage to make it work once. It may be tied to certain windows that don't let you do the scroll, or some other condition. I am currently unsure of the specifics; I will try to pay more attention and update this report if I find a more concrete way to make this issue happen.

### Reproduction Steps

1. Start some action that will create a significant amount of offline entries (for example, a Slayer Task once you have loot stacking, or possibly fishing). The goal here is to have a long list of messages in the summary window.
2. Wait awhile, offline.
3. Resume your game.
4. Attempt to swipe the offline progress toast.

### Expected behaviour

Simply put, it would be expected that I be able to scroll down to the button needed to tap it and thus clear it, allowing me access into the game.

### Which platforms are you experiencing this bug on?

- [ ] Chrome
- [ ] Firefox
- [ ] Edge
- [ ] Safari
- [ ] Steam
- [ ] Mobile App (iOS)
- [ ] Mobile App (Android)
- [X] Mobile App (Android Beta)
- [ ] Other (Please Specify)

### Which version of the game are you experiencing this bug on?

v1.1.2

### Game Subversion

?6099

### Are you using any mods?

Yes
